app/src/main/java/co/nimblehq/compose/crypto/ui/screens/home/HomeScreen.kt

```diff
@@ -22,77 +28,131 @@ import co.nimblehq.compose.crypto.ui.theme.Style.textColor
-                    },
-                text = stringResource(id = R.string.home_title),
-                textAlign = TextAlign.Center,
-                style = Style.semiBold24(),
-                color = MaterialTheme.colors.textColor
-            )
-
-            PortfolioCard(
-                modifier = Modifier
-                    .constrainAs(portfolioCard) {
-                        top.linkTo(title.bottom, margin = Dp40)
-                    }
-                    .padding(horizontal = Dp16)
-            )
+        BoxWithConstraints {
+            val screenHeight = maxHeight
 
-            Text(
+            Column(
                 modifier = Modifier
-                    .constrainAs(myCoinsTitle) {
-                        top.linkTo(portfolioCard.bottom, margin = Dp52)
-                        start.linkTo(parent.start)
-                        width = Dimension.preferredWrapContent
-                    }
-                    .padding(start = Dp16),
-                text = stringResource(id = R.string.home_my_coins_title),
-                style = Style.medium16(),
-                color = MaterialTheme.colors.textColor
-            )
-
-            SeeAll(
-                modifier = Modifier
-                    .clickable(onClick = { /* TODO: Update on Integrate ticket */ })
-                    .constrainAs(seeAll) {
-                        top.linkTo(myCoinsTitle.top)
-                        end.linkTo(parent.end)
-                        width = Dimension.preferredWrapContent
-                    }
-                    .padding(end = Dp16)
-            )
-
-            LazyRow(
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .constrainAs(myCoins) {
-                        top.linkTo(myCoinsTitle.bottom, margin = Dp16)
-                    },
-                contentPadding = PaddingValues(horizontal = Dp16),
-                horizontalArrangement = Arrangement.spacedBy(Dp16),
+                    .fillMaxSize()
+                    .verticalScroll(state = scrollState)
             ) {
-                // TODO: Remove dummy value when work on Integrate.
-                item { CoinItem() }
-                item { CoinItem(true) }
-                item { CoinItem() }
+                Text(
+                    modifier = Modifier
+                        .padding(top = Dp16)
+                        .align(Alignment.CenterHorizontally),
+                    text = stringResource(id = R.string.home_title),
+                    textAlign = TextAlign.Center,
+                    style = Style.semiBold24(),
+                    color = MaterialTheme.colors.textColor
+                )
+
+                PortfolioCard(modifier = Modifier.padding(start = Dp16, top = Dp40, end = Dp16))
+
+                Coins(screenHeight = screenHeight)
+            }
+        }
+    }
+}
+
+@Suppress("FunctionNaming", "LongMethod", "MagicNumber")
+@Composable
+private fun Coins(screenHeight: Dp) {
```

Comment:
@Wadeewee I think this cause the weird behavior on scrolling 🤔 Why do we need to fix the height of the coin list?